### PR TITLE
Prepare release

### PR DESCRIPTION
**Pull request by olemke**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -30,22 +30,27 @@
...
+
+if 'dev' in version:
+    try:
+        cp = subprocess.run(
+            ["git", "describe", "--tags"],
+            stdout=subprocess.PIPE,
+            check=True)
+        so = cp.stdout
+        __version__ = so.strip().decode("ascii").lstrip("v").replace(
+            "-", "+dev", 1).replace("-", ".")
```
Comment (lkluft) [inline setup.py]: You could move these lines into an `else` block

File: typhon/config.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/env python3
```
Comment (lkluft) [inline typhon/config.py:1]: Does it make a difference for a script without execution rights? 😉

File: setup.cfg [hunk 3]
```diff
@@ -1,8 +1,2 @@
-[bdist_wheel]
-# This flag says that the code is written to work on both Python 2 and Python
-# 3. If at all possible, it is good practice to do this. If you cannot, you
-# will need to generate wheels for each Python version that you support.
-universal=1
-
 [aliases]
-test=pytest  # Use pytest to run tests.
+test=pytest
```
Comment (lkluft) [inline setup.cfg:2]: Does ``python setup.py test`` still work without the test runner? Otherwise we could remove the whole file

**Review by lkluft — APPROVED** [review]
In general, the PR looks good 👍 I added some minor comments

File: typhon/config.py (hunk 2 above)
Comment (olemke) [reply typhon/config.py:1]: I'll remove the whole shebang. I don't think it ever makes sense to execute this.

File: setup.cfg (hunk 3 above)
Comment (olemke) [reply setup.cfg:2]: It doesn't...

Comment (olemke) [reply setup.cfg:2]: ...but for other reasons. Using `python setup.py test` fails in the environment test:
```
____________________________________________ ERROR collecting typhon/tests/test_environment.py _____________________________________________
ImportError while importing test module '/Users/u237002/Hacking/git/typhon/typhon/tests/test_environment.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
typhon/tests/test_environment.py:9: in <module>
    from typhon import environ
E   ImportError: cannot import name 'environ' from 'typhon' (/Users/u237002/Hacking/git/typhon/typhon/__init__.py)
```

**Pull request merged**